greenlight/tests/functional/toolbar/test_back_forward.py

```diff
@@ -12,6 +12,14 @@ class TestBackForward(FirefoxTestCase):
     def setUp(self):
         FirefoxTestCase.setUp(self)
 
+        with self.marionette.using_context("content"):
+            self.marionette.navigate("about:blank")
+
+        self.marionette.execute_script("""
+            let count = gBrowser.sessionHistory.count;
+            gBrowser.sessionHistory.PurgeHistory(count);
+        """)
+
```

Comment:
Ok, so lets this go in as is until we have better methods for handling tabs, and setting up and tearing down tests.